franklab_mountainsort/core.py

```diff
@@ -266,6 +267,84 @@ def spike_sort_electrode(animal, date, electrode_number, preprocessing_folder,
                           clip_size=clip_size)
     logger.info(f'{animal} {date} nt{electrode_number} done...')
 
+def recalc_metrics_epoch(raw_mda_file_info,updated_mda='firings_processed.mda',
```

Comment:
It seems like this function could be merged into the function `spike_sort_electrode`

Reply:
updated in pyp.ms4_sort_on_segs()